plutarch-test/src/Plutarch/ScriptsSpec.hs

```diff
@@ -20,9 +20,14 @@ module Plutarch.ScriptsSpec (
 import Data.Text (Text)
 
 import qualified Plutus.V1.Ledger.Api as Plutus
-import qualified Plutus.V1.Ledger.Crypto as Plutus
 
-import Data.Aeson.Extras (encodeSerialise)
+import GHC.Prim (coerce)
```

Comment:
I was under the impression `coerce` can be obtained from `Data.Coerce` as well, which wouldn't require the `ghc-prim` dependency maybe. Could you explain the difference to me please?

Reply:
You're right! I was thinking that too but couldn't remember which it was.